src/Java.Interop/Java.Interop/JavaArray.cs

```diff
@@ -14,7 +14,7 @@ public abstract class JavaArray<T> : JavaObject, IList, IList<T>
 
 		// Value was created via CreateMarshalCollection, and thus can
 		// be disposed of with impunity when no longer needed.
-		protected bool forMarshalCollection;
+		protected bool ForMarshalCollection { get; set; }
```

Comment:
I wonder if this should actually be `internal`. Is there a compelling reason to allow out-of-assembly subclasses to set this?

Reply:
OK, as discussed, all the methods which use it are internal. I will keep it as field and make it internal.